.github/workflows/build.yml

```diff
@@ -0,0 +1,66 @@
+name: Build SDK for latest API version
+
+on: 
+  schedule:
+    # Run on midnight UTC on the first of jan/apr/jul/oct
+    # API is released 1PM EDT/EST
+    - cron: '59 23 1 1/3 *'
+
+jobs:
+  build:
+
+    runs-on: macOS-latest
+    
+    steps:
+    - name: Get API Version
+      id: version
+      run: |
+        API_VERSION="$(date +'%Y-%m')"
+        echo "::set-env name=API_VERSION::$API_VERSION"
+      
+    - uses: actions/checkout@master
+      with:
+        ref: 'develop' # This should change to master after 2020-04
```

Comment:
Can we change this to master now and clean up the comment? Or would you rather open another PR to follow this